src/auth/googleauth.ts

```diff
@@ -87,6 +87,11 @@ export class GoogleAuth {
    */
   static DefaultTransporter = DefaultTransporter;
 
+  /**
+   *  @param {number=} refreshTokenEarlyMillis The token should be refreshed if it will expire within this many milliseconds.
```

Comment:
Omit JSDoc type annotations in TypeScript code

Reply:
Done.